src/Components/Publishing/ReadMore/ReadMoreButton.tsx

```diff
@@ -1,20 +1,34 @@
 import { unica } from "Assets/Fonts"
+import { once } from "lodash"
 import React from "react"
+import Waypoint from "react-waypoint"
 import styled from "styled-components"
 import { track } from "../../../Utils/track"
 import { pMedia } from "../../Helpers"
 import { StandardLayoutParent } from "../Layouts/StandardLayout"
 
-@track()
-export class ReadMore extends React.Component<any, any> {
-  constructor(props) {
-    super(props)
-    this.onClick = this.onClick.bind(this)
+interface ReadMoreProps {
+  onClick: () => void
+  tracking?: any
+}
+
+export class ReadMore extends React.Component<ReadMoreProps, any> {
```

Comment:
you can just do `React.Component<ReadMoreProps>` without including any for State